fr-fr.json

```diff
@@ -232,27 +232,27 @@
     "Dashboard": {
         "views": "Vus",
         "likes": "J'aime",
-        "addPlaylists": "Ajouter à la playlist",
-        "newPlaylist": "Nouvelle playlist",
+        "addPlaylists": "Ajouter à la Bibliothèque",
+        "newPlaylist": "Nouvelle Bibliothèque",
         "addPlaylistButton": "Ajouter",
         "addPLaylistTitle":"Ajouter à...",
         "live":"Go",
-        "ago": "ago"
+        "ago": "il y a"
```

Comment:
This goes before time indicication in french.
En: 3 minutes ago
fr: Il y a 3 minutes